### PR TITLE
Fix `FieldDefinitionNode.arguments` for graphql

### PR DESCRIPTION
**Pull request by sullvn**
- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/graphql/graphql-js/blob/787422956c9554d12d063a41fe35705335ec6290/src/language/printer.js#L139
- [x] Increase the version number in the header if appropriate.
- [x] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

File: types/graphql/language/ast.d.ts [hunk 1]
```diff
@@ -424,7 +424,7 @@ export interface FieldDefinitionNode {
     readonly loc?: Location;
     readonly description?: StringValueNode;
     readonly name: NameNode;
-    readonly arguments?: ReadonlyArray<InputValueDefinitionNode>;
+    readonly arguments: ReadonlyArray<InputValueDefinitionNode>;
```
Comment (firede) [inline types/graphql/language/ast.d.ts]: It seems that `arguments` is an optional value.
https://github.com/graphql/graphql-js/blob/master/src/language/ast.js#L444

**Review by firede — CHANGES_REQUESTED** [review]
Thank you for your PR!
`@types/graphql` should probably follow the definition of `graphql-js`.
If you find that the implementation has a bug, or you need to modify the interface, please give [them](https://github.com/graphql/graphql-js) a PR.

File: types/graphql/language/ast.d.ts (hunk 1 above)
Comment (awfulaxolotl) [reply types/graphql/language/ast.d.ts]: Ah! Thanks for the quick response.

It seems like `graphql-js` has a bug in the AST printer then:

https://github.com/graphql/graphql-js/blob/787422956c9554d12d063a41fe35705335ec6290/src/language/printer.js#L139

It fails if `arguments` is indeed undefined. I'll make a PR on their end so it handles it correctly.

**Comment by awfulaxolotl** [comment]
I've submitted a PR to fix the implementation instead of the types. I'll close this PR if that becomes accepted.

PR: https://github.com/graphql/graphql-js/pull/1548

**Pull request closed without merging**